PINCE.py

```diff
@@ -1942,6 +1953,25 @@ def treeWidget_AddressTable_edit_value(self):
                 debugcore.write_memory(address, vt.value_index, parsed_value, vt.zero_terminate, vt.endian)
             self.update_address_table()
 
+    def treeWidget_AddressTable_edit_offset(self):
+        row = guiutils.get_current_item(self.treeWidget_AddressTable)
+        if not row:
+            return
+
+        dialog = InputDialogForm(self, item_list=[("Offset addresses by:",'')])
+        if dialog.exec():
+            offset_value = dialog.get_values()
+            offset_int = int(offset_value, 16)
+            print(offset_value)
+            print(offset_int)
+            for row in self.treeWidget_AddressTable.selectedItems():
+                desc, address_expr, value_type = self.read_address_table_entries(row)
+                address = row.text(ADDR_COL).strip("P->")
```

Comment:
You're stripping P-> but that's grabbing the resolved address from the pointer chain.
Unless you modify the chain and add the value to the last offset directly, it will just break again.